controllers/depresolver/depresolver_test.go

```diff
@@ -1198,7 +1198,7 @@ func TestResolveLoggerNoColorInvalidValue(t *testing.T) {
 	config, err := resolver.ResolveOperatorConfig()
 	// assert
 	assert.NoError(t, err)
-	assert.Equal(t, true, config.Log.NoColor)
+	assert.Equal(t, false, config.Log.NoColor)
```

Comment:
Could you please explain the semantics of this test?  It is invalid, still intentionally NoError?

Reply:
yes, it is expected that NoColor resolution falls back to `false` without error, when invalid value is used for NO_COLOR env var.